gram.y

```diff
@@ -478,6 +483,96 @@ v6addrlist_rasrcaddress	: IPV6ADDR ';'
 		}
 		;
 
+nat64prefixdef	: nat64prefixhead optional_nat64prefixplist ';'
+		{
+			if (nat64prefix) {
+
+				if (nat64prefix->AdvValidLifetime > 65528)
```

Comment:
Please move this constant elsewhere, and document that it comes from the RFC

Reply:
Done